docs/guides/component-testing/component-framework-configuration.mdx

```diff
@@ -45,26 +70,34 @@ resolves to a `devServer` instance.
 
 :::
 
-## Custom Index File
-
-By default, Cypress renders your components into an HTML file located at
-`cypress/support/component-index.html`.
+Any requests triggered during a test using the `devServerPublicPathRoute` as
+defined in the `cypressConfig` will be forwarded to your server. The
+cypress-runner will trigger a request for an index.html when a test is started.
+Your server needs to send the custom index.html file after injecting a script to
+load the actual test. If you do not bundle your support file along with the
+tests, you need to add a separate import statement for the support file.
```

Comment:
```suggestion
Any requests triggered during a test using the `devServerPublicPathRoute` as
defined in the `cypressConfig` will be forwarded to your server. Cypress
will trigger a request for `component-index.html` when a test is started.
Your server needs to reply with a custom `index.html` file after injecting a script to
load the actual test. If you do not bundle your support file along with the
tests, you need to add a separate import statement for the support file.
```

Might be worth linking to an example in the Cypress code base, so third parties have something to reference. What do you think?

Reply:
Sounds good, I wasn't sure if this is good practice for the docs.